starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -652,10 +652,41 @@ sub check_dither {
         }
     }
 
+    # Seed the max dither amplitude with the amplitude 8 minutes into the observation
+    $guide_dither->{ampl_y_max} = $guide_dither->{ampl_y};
+    $guide_dither->{ampl_p_max} = $guide_dither->{ampl_p};
+
+    # Check for dither changes during the observation.
+    # This also updates the maximum amplitude for any observation that has dither
+    # changes.
+    # ACA-003
```

Comment:
This chunk is moved not new - I just thought the check for changes should happen earlier - though explicitly this is the only check that *uses* the maximum amplitude of dither if there are changes during the observation as well.